pkg/systemd/services/services.jsx

```diff
@@ -70,8 +70,8 @@ export function updateTime() {
                 clock_monotonic_now = parseInt(uptime * 1000000, 10);
             }, ex => console.log(ex.toString()));
     cockpit.spawn(["date", "+%s"])
-            .then(function(time) {
-                clock_realtime_now = moment.unix(parseInt(time));
+            .then(time => {
+                clock_realtime_now = parseInt(time, 10) * 1000;
```

Comment:
explanation: moment.unix is moment(timestamp * 1000)

I had to check this one.